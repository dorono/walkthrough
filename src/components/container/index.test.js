import React from 'react';
import {shallow} from 'enzyme';
import Container from './index';

describe('Container', () => {
    it('should render without errors', () => {
        shallow(<Container />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Container />)).toMatchSnapshot();
    });
});
