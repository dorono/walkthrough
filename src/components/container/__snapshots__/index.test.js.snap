// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match snapshot 1`] = `
<div
  className="root section"
>
  <header
    className="header"
  />
  <AlertBarLegend
    highlightBanner={true}
    show={false}
  />
</div>
`;
